moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -58,6 +58,7 @@
 #include <tf2_ros/buffer.h>
 
 #include <boost/python.hpp>
+#include <boost/noncopyable.hpp>
```

Comment:
remove?

Reply:
But why would it have to be added back? we don't use noncopyable, do we?

Reply:
Later in this file there is `bp::class_<MoveGroupInterfaceWrapper, boost::noncopyable> move_group_interface_class(
      "MoveGroupInterface", bp::init<std::string, std::string, bp::optional<std::string, double>>());`